help/network-issues.md

```diff
@@ -2,4 +2,32 @@
 description: >-
   Describes the Network incident in details and possibles causes. Also describes some common best practices
   in setting up network for accessing Office 365.
----
\ No newline at end of file
+---
+
+**Network issues** is an incident that is raised for a location when SysKit Sense determines that the network is behaving poorly loading sites from that location.

```

Comment:
Not sure about "is behaving poorly loading sites", maybe "is behaving poorly during site loading process"? What do you think ?

Reply:
changed it to '... site load'